examples/access-control/tests/access_control.rs

```diff
@@ -32,7 +31,7 @@ async fn deploy(account: &Account) -> eyre::Result<Address> {
 #[e2e::test]
 async fn constructs(alice: Account) -> Result<()> {
     let alice_addr = alice.address();
-    let contract_addr = deploy(&alice).await?;
+    let contract_addr = deploy(&alice, constructor()).await?.address()?;
```

Comment:
Also it could look like: `let contract_addr = alice.deploy(constructor()).await?.address()?;`
Or `let contract_addr = alice.deploy_with(constructor()).await?.address()?;`